pkg/middleware/auth.go

```diff
@@ -64,10 +64,15 @@ func writeRedirectCookie(c *models.ReqContext) {
 		redirectTo = setting.AppSubUrl + c.Req.RequestURI
 	}
 
-	// remove any forceLogin=true params
-	redirectTo = removeForceLoginParams(redirectTo)
+	if redirectTo != "/" {
+		if setting.AppSubUrl != "" && !strings.HasPrefix(redirectTo, setting.AppSubUrl) {
+			redirectTo = setting.AppSubUrl + c.Req.RequestURI
+		}
```

Comment:
Address this before merging: This looks like a repeat of lines 63-65, I don't think you need these lines?

```suggestion
```

Reply:
Oops! I missed this, thank you!